lib/ui/huge_listview/lazy_loading_gallery.dart

```diff
@@ -352,7 +352,7 @@ class _LazyLoadingGridViewState extends State<LazyLoadingGridView> {
 
   Widget _getRecyclableView() {
     return VisibilityDetector(
-      key: UniqueKey(),
+      key: Key(widget.filesInDay.first.tag),
```

Comment:
Can you try archive or delete all files in a day and confirm that it doesn't break anything?

Reply:
Archive worked as expected.